code/worker.sh

```diff
@@ -3,8 +3,8 @@
 URL=$(cat $1 | head -n 1)
 echo $URL
 
-if [ x"$TRAVIS_PULL_REQUEST" == xfalse ] ; then
-  git checkout "$TRAVIS_BRANCH"
+if [ x"$GITHUB_ACTIONS" == xfalse ] ; then
```

Comment:
I'm not too sure about these 2 vars and their Travis/Actions equivalent. Suggestions?